tools/mentors/view_page_text_image.php

```diff
@@ -6,6 +6,10 @@
 
 require_login();
 
+$projectid = array_get($_GET, "project", "");
```

Comment:
This should use `validate_projectID()` (although I have an MR out where it would be `get_projectID_param()`).

Reply:
I'll use your new functions.